### ISSUE TITLE
[New Package]: Appest.TickTick

### ISSUE THREAD

**Issue by SpecterShell** [vedantmgoyal2009/vedantmgoyal2009#110]
### List the PackageIdentifier of the package to be added to automation

Appest.TickTick

### Details about the application

1. Use `$result.version` to get and check the version
   https://pull.ticktick.com/windows/release_note.json
2. Get the installer URL from download link, along with getting installer file
   If only the url is needed, you can use `HEAD` method for webrequest to avoid fetching full response.
   ```powershell
   $webrequest = Invoke-WebRequest 'https://ticktick.com/static/getApp/download?type=win64'
   $installerurl = $webrequest.BaseResponse.RequestMessage.RequestUri.OriginalString
   ```

**Comment by SpecterShell** [comment]
This one is not working properly. https://github.com/microsoft/winget-pkgs/pull/41602
Maybe it's my fault that I forgot to mention there are two installers.

```PowerShell
Invoke-WebRequest -Uri 'https://ticktick.com/static/getApp/download?type=win64' -Method Head
Invoke-WebRequest -Uri 'https://ticktick.com/static/getApp/download?type=win' -Method Head
```
BTW, it is recommended to use `-Method Head` after `iwr` as it can avoid downloading file and save a lot of time.

**Comment by vedantmgoyal2009** [comment]
Fixed in Automation v2.0.0.

**Comment by SpecterShell** [comment]
https://github.com/vedantmgoyal2009/vedantmgoyal2009/issues/259
...so does this one.
